capstone/src/main/webapp/statistics_script.js

```diff
@@ -118,42 +123,66 @@ async function getSurveyResponses() {
   loadAgePieChart(ageCount);
 }
 
-function loadBubbleChart(days, scores, genders, responseTimeSThree) {
-  const stats = new google.visualization.DataTable();
+// Default global values
+let directExperience = 'No';
+let gender = 'Male';
+let ageRange = '18-24';
+
+async function getPrediction(choice, category) {
+  if (category === 'gender') {
+    gender = choice.options[choice.selectedIndex].text;
+  } else if (category === 'ageRange') {
+    ageRange = choice.options[choice.selectedIndex].text;
+  } else if (category === 'directExperience') {
+    directExperience = choice.options[choice.selectedIndex].text;
+  }
 
-  stats.addColumn('string', 'ID');
-  stats.addColumn('number', 'Day in July');
-  stats.addColumn('number', 'Sentiment Score');
-  stats.addColumn('string', 'Gender');
-  stats.addColumn('number', 'Response time (in milliseconds)');
+  // Ensures values match those in data store
+  if (directExperience ==='Unknown') {
+    directExperience = 'NoResponse';
+  }
 
-  // All parameters are the same length
-  const listLength = days.length;
-  const res = 'Response #';
+  if (gender ==='Unknown') {
+    gender = 'UnknownGender';
+  }
 
-  for (let i = 0; i < listLength; i++) {
-    stats.addRows([
-      [res + i.toString(10),
-        days[i], scores[i], genders[i], responseTimeSThree[i]],
-    ]);
+  if (ageRange ==='Unknown') {
+    ageRange = 'UnknownAge';
   }
```

Comment:
Why do this extra work? Could you have the values in datastore match these 'Unknown' values instead?

Reply:
Since I encode each variable in the python program for the model, they need to have unique values. I was not able to assign Unknown to each of the categories so I had to make adjustments